wagtail/wagtailadmin/edit_handlers.py

```diff
@@ -6,7 +6,7 @@
 from six import text_type
 
 from modelcluster.forms import ClusterForm, ClusterFormMetaclass
-
+from modelcluster.models import get_related_model
```

Comment:
Would probably be best to redefine this in wagtail so an older version of modelcluster can still be used under Django 1.7